Tasks/KubernetesV1/task.loc.json

```diff
@@ -46,7 +46,7 @@
       "name": "advanced",
       "displayName": "ms-resource:loc.group.displayName.advanced",
       "isExpanded": false,
-      "visibleRule": "command != login && command != logout"
+      "visibleRule": "command != logout"
```

Comment:
Should we remove this as well? Can you please get it reviewed from vithati?

Reply:
Discussed with Anumita. We will show advanced options for both login and logout.